src/utils/duration.js

```diff
@@ -39,6 +40,22 @@ function toDoubleDigits(num) {
     }
 }
 
+function addTimeSuffix(value, suffix) {
+    return typeof value === 'number' && value > 0 ? `${value + suffix}` : '';
+}
+
+export function milisecondsToDHMS(numericDuration) {
+    const ms = numericDuration || 0;
+    const dhms = [
+        addTimeSuffix(Math.floor(normalizeAge(ms / ONE_DAY)), 'd'),
+        addTimeSuffix(Math.floor(normalizeAge((ms % ONE_DAY) / ONE_HOUR)), 'h'),
+        addTimeSuffix(Math.floor(normalizeAge((ms % ONE_HOUR) / ONE_MINUTE)), 'm'),
+        addTimeSuffix(Math.floor(normalizeAge((ms % ONE_MINUTE) / ONE_SECOND)), 's')
+    ].filter(Boolean).join(' ');
```

Comment:
Can we remove this filter? I don't think it's necessary. Otherwise, looks great!

Reply:
Added missing an extra "l" in the method name - should be 'millisecondsToDHMS' instead of 'milisecondsToDHMS'.

I've added the missed extra 'l'. As to the array.filter(Boolean), the filter is the same as array.filter(i => Boolean(i)) which in this case, filters some of DHMS elements that are empty string and generate extra space during joining.

Reply:
Ah... gotcha! Nice! I'm going to get one more set of eyes on this as the formatting for UTC is widely used. Otherwise looks great!